include/zephyr/bluetooth/conn.h

```diff
@@ -1849,12 +1856,16 @@ struct bt_conn_cb {
 	 * This callback notifies the application that it might be able to
 	 * allocate a connection object. No guarantee, first come, first serve.
 	 *
-	 * Use this to e.g. re-start connectable advertising or scanning.
+	 * The maximum number of simultaneous connections is configured
+	 * by @kconfig{CONFIG_BT_MAX_CONN}, which usually defaults to a single connection.
```

Comment:
```suggestion
	 * The maximum number of simultaneous connections is configured
	 * by @kconfig{CONFIG_BT_MAX_CONN}.
```

I don't think the second part of this sentence is helpful or necessary. I'd rather just refer to the Kconfig option's default value

Reply:
Okay, I'll remove this second part.